system/lib/libc/emscripten_time.c

```diff
@@ -19,8 +19,8 @@ __attribute__((__weak__)) int   daylight = 0;
 __attribute__((__weak__)) char *tzname[2] = { 0, 0 };
 
 void _tzset_js(long* timezone, int* daylight, char** tzname);
-time_t _timegm_js(struct tm *tm);
-time_t _mktime_js(struct tm *tm);
+int _timegm_js(struct tm *tm);
+int _mktime_js(struct tm *tm);
```

Comment:
Maybe comment here as to why these are different?

Reply:
Done